lib/stuff.php

```diff
@@ -142,19 +142,21 @@ function require_feature_files() {
  * Launches a new WordPress instance on the managed server
  *
  * @param  string $php_version          The PHP version to run the app on.
- * @param  array  $requested_features   Array of features to enable.
- *        boolean config-constants      Should we add the Config Constants plugin to the site?
- *        boolean auto_ssl              Should we add Let's Encrypt-based SSL for the site?
- *        boolean ssl                   Should we add the configured SSL certificate for the site?
- *        boolean gutenberg             Should we add Gutenberg to the site?
- *        boolean jetpack               Should we add Jetpack to the site?
- *        boolean jetpack-beta          Should we add Jetpack Beta Tester plugin to the site?
- *        boolean subdir_multisite      Should we enable subdir-based multisite on the site?
- *        boolean subdir_multisite      Should we enable subdomain-based multisite on the site?
- *        boolean woocommerce           Should we add WooCommerce plugin to the site?
- *        boolean wordpress-beta-tester Should we add Jetpack Beta Tester plugin to the site?
- *        boolean wp-debug-log          Should we set WP_DEBUG and WP_DEBUG log to true ?
- *        boolean wp-log-viewer         Should we add WP Log Viewer plugin to the site.
+ * @param  array  $requested_features {
```

Comment:
Linter was complaining, so I end up updating the whole chunk to match WP standard: https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#1-1-parameters-that-are-arrays

Reply:
nice